assets/js/modules/analytics-4/datastore/audience-settings.test.js

```diff
@@ -52,23 +55,31 @@ describe( 'modules/analytics-4 audience settings', () => {
 	);
 
 	beforeEach( () => {
-		// To-Do: After #8888 is implemented, we can directly use createTestRegistry from 'tests/js/utils'
+		// TODO: After #8888 is implemented, we can directly use `createTestRegistry` from 'tests/js/utils',
+		// rather than registering the stores below.
 		registry = createRegistry();
 
 		registry.registerStore(
 			MODULES_ANALYTICS_4,
-			combineStores( AudienceSettingsStore, commonStore, {
-				...AudiencesStore,
+			combineStores( audienceSettingsStore, commonStore, {
+				audiencesStore,
 				resolvers: {},
+				actions: {
+					*syncAvailableAudiences() {
```

Comment:
Let's add that comment I suggested, I'll do this myself to save a round trip:

```suggestion
					// Provide a minimal implementation of `syncAvailableAudiences` for testing.
					*syncAvailableAudiences() {
```